Makefile

```diff
@@ -30,6 +30,9 @@ PKGER ?= ./hack/update-pkger.sh
 CODE := $(shell find . -name '*.go') pkged.go go.mod schema/func_yaml-schema.json
 TEMPLATES := $(shell find templates -name '*' -type f)
 
+# Code for goimports (checkstyle)
+CODECHECK := $(shell find . -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -name 'wire_gen.go' -prune -o -type f -name '*.go' -print)
```

Comment:
Please help me understand:  Why is this set of files different than the set of files already used for checks (the CODE variable above)?

Perhaps the correct solution here is to use the CODE variable above, and modify it as necessary to include any of the additional `find` options that you see as necessary for running code checks.

After all, why would we "codechec" files that are not "code" used elsewhere?

Please let me know if you don't think this would make sense for any reason, but perhaps ti would be better to use the CODE files above, and augment that `find` command with the conditions of omitting the `vendor` directory, protobuffs and wire_gen?